src/components/FindWallet/WalletTable.tsx

```diff
@@ -541,8 +541,6 @@ const WalletTable = ({ data, filters, walletData }) => {
           onChange={(selectedOption) => {
             setFirstFeatureSelect(selectedOption)
           }}
-          defaultValue={firstFeatureSelect}
```

Comment:
We should add these back in: https://www.figma.com/file/VfAtok8dxIxQf6hYA4Cghu?node-id=1694:31752#214197804